src/component/Validation.js

```diff
@@ -1,135 +1,126 @@
-import React, {Component} from 'react';
+import React, {useState, useEffect} from 'react';
 import AppBar from '@material-ui/core/AppBar';
-import Button from '@material-ui/core/Button' //<Button variant="contained"
+import Button from '@material-ui/core/Button'
 import Typography from "@material-ui/core/Typography";
-import { ValidatorForm, TextValidator } from 'react-material-ui-form-validator';
+import {ValidatorForm, TextValidator} from 'react-material-ui-form-validator';
 
 
-class UserDetail extends Component {
-    state = {
-        formData: {
-            firsName: '',
-            lastName: '',
-            email: '',
-            password: '',
-            repeatPass: ''
-        },
-        submitted: false,
-    }
-    componentDidMount() {
-        // Password match
+let UserDetail = () => {
```

Comment:
You should better use `const` instead of `let`

Reply:
Done.